apis/fluentbit/v1alpha2/plugins/output/loki_types.go

```diff
@@ -19,11 +19,16 @@ type Loki struct {
 	// +kubebuilder:validation:Minimum:=1
 	// +kubebuilder:validation:Maximum:=65535
 	Port *int32 `json:"port,omitempty"`
+	// Specify a custom HTTP URI. It must start with forward slash.
+	Uri string `json:"uri,omitempty"`
```

Comment:
nit: is it better to use *string along with `omitempty ` here to make it consistent with other fields?